assets/js/components/settings/SettingsActiveModule/ConfirmDisconnect.js

```diff
@@ -153,7 +153,7 @@ export default function ConfirmDisconnect( { slug } ) {
 				'google-site-kit'
 			),
 			name,
-			listFormat( dependentModules )
```

Comment:
Why was this removed?

Reply:
Huh, that's odd, not sure? I missed that in the huge PR, my bad 😞

Reply:
Might've been a bad merge actually, looking at things. 🤔 Can't see a reason why it'd be removed otherwise. 🤷🏻‍♂️